java/overrides.js

```diff
@@ -50,6 +53,10 @@
     'appium android check window': {skip: true}, //wrong ignore region ticket 2396
     'appium android check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
     'appium iOS check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
+	
+	'check window fully on page with horizontal scroll with css stitching': {skip: true}, //Could not start a new session
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with css stitching": {skip: true}, //Could not start a new session
```